frontend/app/components/Session_/BottomBlock/bottomBlock.module.css

```diff
@@ -4,6 +4,4 @@
   /* padding-right: 10px; */
   /* border: solid thin $gray-light; */
   height: 300px;
```

Comment:
I don't have a strong opinion on look, so if I modified something somewhere, it's very likely because it's inconsistent (this padding here is re-applied elsewhere, leading to double padding in the information header). 

The border removal is accidental. I rebased on a 7 month old dev branch first, where the border didn't exist I think.